openedx/features/discounts/applicability.py

```diff
@@ -143,7 +143,7 @@ def _is_in_holdback(user):
     """
     Return whether the specified user is in the first-purchase-discount holdback group.
     """
-    if datetime(2020, 8, 1, tzinfo=pytz.UTC) <= datetime.now(tz=pytz.UTC):
+    if datetime(2021, 8, 1, tzinfo=pytz.UTC) <= datetime.now(tz=pytz.UTC):
```

Comment:
It won't work after 2021. One way is to get the current year and add 1 year into it.

Reply:
changed to current year